test/remote/gateways/remote_mercury_test.rb

```diff
@@ -46,13 +46,13 @@ def test_successful_authorize_and_capture
   end
 
   def test_failed_authorize
-    response = @gateway.authorize(1100, @credit_card, @options)
+    response = @gateway.authorize(257, @credit_card, @options)
```

Comment:
is this a `magic number` or can it be `@amount`?  If it is a magic number, can we set it as a variable instead of using the magic number here?

Reply:
It's magic number it triggers the Decline message but I'll create a variable for it